lsp/src/lsp-autocomplete.c

```diff
@@ -315,6 +315,51 @@ LspAutocompleteSymbol *find_symbol(GeanyDocument *doc, const gchar *text)
 }
 
 
+#if ! GLIB_CHECK_VERSION (2, 68, 0)
+static guint lsp_g_string_replace(GString *string, const gchar *find, const gchar *replace, guint limit)
```

Comment:
Yes, I re-implemented that manually out of sheer 1am :laughing: You might want to use a copy of the GLib implementation, although this one should work fine.